pkg/apply/task/inv_set_task_test.go

```diff
@@ -134,7 +134,7 @@ func TestInvSetTask(t *testing.T) {
 				t.Errorf("unexpected error running InvAddTask: %s", result.Err)
 			}
 			actual, _ := client.GetClusterObjs(nil, common.DryRunNone)
-			if !object.SetEquals(tc.expectedObjs, actual) {
+			if !object.IsSetEqual(tc.expectedObjs, actual) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.